x-pack/test/functional/apps/rollup_job/hybrid_index_pattern.js

```diff
@@ -0,0 +1,106 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+
+import datemath from '@elastic/datemath';
+import expect from '@kbn/expect';
+import mockRolledUpData, { mockIndices } from './hybrid_index_helper';
+
+export default function ({ getService, getPageObjects }) {
+
+  const es = getService('es');
+  const esArchiver = getService('esArchiver');
+  const retry = getService('retry');
+  const PageObjects = getPageObjects(['common', 'settings']);
+
+  describe('hybrid index pattern', function () {
+    //Since rollups can only be created once with the same name (even if you delete it),
+    //we add the Date.now() to avoid name collision.
```

Comment:
Nit: should we be explicit and clarify that this matters if you run the tests back to back? Otherwise the reader needs to infer this intention.

Reply:
Done.